### DISCUSSION TITLE
Import/Export with subtable config

### DISCUSSION THREAD

**Discussion by st-trade** [AntlerVC/firetable#860] [Idea / Feature Request]
I'd like to ask for table structure replication for subcollections/subtables.

Currently, each subtable in a record has to be initialized manually, since it's impossible to import table structure from other places. But it would be great if this schema was synced automatically.  Say, the same table structure for all subtables in a particular table.
<img width="1075" alt="image" src="https://user-images.githubusercontent.com/94030557/195175750-d4f964f7-adb8-4000-b033-6e16549404c4.png">

**Comment by harinij** [comment]
@st-trade thanks for this suggestion, we will add it to our list of items to work on.

**Comment by harinij** [comment]
Hi @st-trade you can now import/export table with subtable config as part of the table creation process as part of [this PR](https://github.com/rowyio/rowy/pull/1066)
